drivers/SmartThings/sonos/src/init.lua

```diff
@@ -1,26 +1,12 @@
 SONOS_API_KEY = require 'app_key'
 local old_log = require "log"
 
--- Forward all debug logs to hubcore info logs.
-old_log.debug = function(...)
-  old_log.info_with({hub_logs = true}, table.concat({"[DEBUG->INFO] ", ...}))
-end
-
--- Print all errors to hubcore for now
-old_log.error = function(...)
-  old_log.error_with({hub_logs = true}, ...)
```

Comment:
I would prefer an approach where we can leave all error and warning logs going to hub logs. If we think the error/warning wouldn't actually be informative during an investigation or it would be too chatty, we should remove the log or reduce its level. As we solve crashes that are occuring on errors https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/841, I would like to ensure we still have some way to get insight to if errors are occuring.

Reply:
I'm actually in support of leaving the driver-global error and warning overrides here in place for Warning and Error as well.

Reply:
Added driver-global error and warning logging overrides in cb1f83f1520e03b2374e7b07665335d85e6a76fe